File: C#/프로그래머스/0/181853. 뒤에서 5등까지/뒤에서 5등까지.cs
using System;
using System.Linq;
using System.Collections.Generic;

public class Solution {
    private int COUNT_RESULT = 5;
    // num_list의 원소가 겹치지 않는다는 보장(제한조건)이 없음
    // 정렬을 수정하거나, 원소를 지워나가는 방식으로 해야 함.
    public int[] solution(int[] num_list) {
        
        List<int> n_list = num_list.ToList();
        int[] answer = new int[COUNT_RESULT];
        
        for(int i = 0; i < COUNT_RESULT; i++) {
            int min_idx = 0;
            for(int j = 0; j < n_list.Count; j++) {
                if (n_list[min_idx] > n_list[j]) {
                    min_idx = j;
                }
            }
            answer[i] = n_list[min_idx];
            n_list.RemoveAt(min_idx);
        }
        
        return answer;
    }
}

File: Python3/프로그래머스/0/181853. 뒤에서 5등까지/뒤에서 5등까지.py
def solution(num_list):
    n_list = num_list[:]
    answer = []
    
    for i in range(5):
        min_idx = 0
        for j in range(len(n_list)):
            if n_list[min_idx] > n_list[j]:
                min_idx = j
        answer.append(n_list[min_idx])
        del n_list[min_idx]
    
    return answer